src/Common/APIInterfaceRule.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace ArchitectureSniffer\Common;
+
+use ArchitectureSniffer\SprykerAbstractRule;
+use PHPMD\AbstractNode;
+use PHPMD\Node\MethodNode;
+use PHPMD\Rule\InterfaceAware;
+
+class APIInterfaceRule extends SprykerAbstractRule implements InterfaceAware
```

Comment:
Do we anywhere use all uppercase for acronyms?
We usually always PascalCase (`Api`Interface) these things in class names.